solutions/swb-app/src/projectRoutes.ts

```diff
@@ -3,16 +3,37 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { ProjectService, CreateProjectRequest, ListProjectsRequest } from '@aws/workbench-core-accounts';
+import {
+  ProjectService,
+  CreateProjectRequest,
+  ListProjectsRequest,
+  GetProjectRequest
+} from '@aws/workbench-core-accounts';
 import { AuthenticatedUser } from '@aws/workbench-core-authorization';
 import { Request, Response, Router } from 'express';
 import { validate } from 'jsonschema';
 import { wrapAsync } from './errorHandlers';
 import CreateProjectSchema from './schemas/projects/createProjectSchema';
+import GetProjectSchema from './schemas/projects/getProjectSchema';
 import ListProjectsSchema from './schemas/projects/listProjectsSchema';
 import { processValidatorResult } from './validatorHelper';
 
 export function setUpProjectRoutes(router: Router, projectService: ProjectService): void {
+  // Get project
+  router.get(
+    '/projects/:id',
```

Comment:
nit: if we decide to provide more granular routes ( `/users/userId/projects/projectId` ), it would be better to have more descriptive parameters. id -> projectId

Reply:
will change! thanks for the suggestion!